person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..200)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    of_age? || parent_permission
```

Comment:
- [ ] We should use `@parent_permission` instance variable, instead of `parent_permission`.